scripts/docker-ci-entrypoint.sh

```diff
@@ -4,9 +4,16 @@ set -euxo pipefail
 
 export CI=true
 
-cp -r ../package/* .
+# enable extended globbing for omitting build artifacts
+shopt -s extglob
 
-cp ../root/yarn.lock .
+# avoid copying build artifacts from the host
+cp -r ../package/!(node_modules|dist|dist-cjs|dist-esm|build|.next|.turbo) .
+
+# copy cache
```

Comment:
There's no way that someone could push a modified cache, causing the build to pull in weird packages, right?

Reply:
this should either use the local instance of cache or the cache provided by Github Actions. I don't think this PR increases the attack surface, as devs can add dependencies already and we would need to manually allow running workflows.